src/index.js

```diff
@@ -48,4 +48,6 @@ var namespaces         = 0
 	return getter;
 }
 
-module.exports = { getNamespace, getCountNamespaces, startRequest, bind, patch };
+module.exports = global.requestLocalStorage || { getNamespace, getCountNamespaces, startRequest, bind, patch };
+
+if (!global.requestLocalStorage) global.requestLocalStorage = module.exports;
```

Comment:
Maybe we could pull in @doug-wade's change along with the versioning? I'll update the PR.